extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.js

```diff
@@ -85,6 +86,28 @@ ve.init.wikia.ViewPageTarget.static.actionsToolbarConfig = [
 	}
 ];
 
+/**
+ * @inheritdoc
+ */
+ve.init.wikia.ViewPageTarget.prototype.setSurface = function ( surface ) {
+	// Parent method
+	ve.init.mw.ViewPageTarget.prototype.setSurface.call( this, surface );
+	this.setupLicense( surface );
+};
+
+/**
+ * @inheritdoc
+ */
+ve.init.wikia.ViewPageTarget.prototype.tearDownSurface = function ( noAnimate ) {
+	if ( this.$license ) {
+		this.$license.remove();
+		this.$license = null;
+	}
+
+	// Parent method
+	return ve.init.mw.ViewPageTarget.prototype.tearDownSurface.call( this, noAnimate );
+};
+
 //ve.init.wikia.ViewPageTarget.static.surfaceCommands.push( 'wikiaSourceMode' );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)